apps/studio/components/interfaces/Realtime/Inspector/ChooseChannelPopover/index.tsx

```diff
@@ -83,14 +94,18 @@ export const ChooseChannelPopover = ({ config, onChangeConfig }: ChooseChannelPo
           {config.channelName.length === 0 ? (
             <>
               <Form_Shadcn_ {...form}>
-                <form id="realtime-channel" onSubmit={form.handleSubmit(() => onSubmit())}>
+                <form
+                  id="realtime-channel"
+                  onSubmit={form.handleSubmit(() => onSubmit())}
+                  className="flex flex-col gap-y-4"
+                >
                   <FormField_Shadcn_
                     name="channel"
                     control={form.control}
                     render={({ field }) => (
                       <FormItem_Shadcn_>
```

Comment:
to give some spacing between the label and the input
```suggestion
                      <FormItem_Shadcn_ className="flex flex-col gap-y-1>
```